### PR TITLE
Docs for RMAT

### PR DESCRIPTION
**Pull request by BradReesWork**
Fixed indentations so that docs appear correctly

### CODE REVIEW

File: python/cugraph/generators/rmat.py [hunk 1]
```diff
@@ -243,49 +243,68 @@ def rmat(
     Parameters
     ----------
     scale : int
-    Scale factor to set the number of verties in the graph Vertex IDs have
-    values in [0, V), where V = 1 << 'scale'
+        Scale factor to set the number of verties in the graph Vertex IDs have
```
Comment (rlratzel) [inline python/cugraph/generators/rmat.py:246]:
```suggestion
        Scale factor to set the number of vertices in the graph Vertex IDs have
```

File: python/cugraph/generators/rmat.py [hunk 2]
```diff
@@ -243,49 +243,68 @@ def rmat(
...
-    Probability of the second partition
+        Probability of the second partition
 
     c : float
-    Probability of the thrid partition
+        Probability of the thrid partition
 
     seed : int
-    Seed value for the random number generator
+    S   eed value for the random number generator
```
Comment (rlratzel) [inline python/cugraph/generators/rmat.py:262]:
```suggestion
    Seed value for the random number generator
```

File: python/cugraph/generators/rmat.py [hunk 3]
```diff
@@ -243,49 +243,68 @@ def rmat(
...
+        Flag controlling whether to generate edges only in the lower triangular
+        part (including the diagonal) of the graph adjacency matrix
+        (if set to 'true') or not (if set to 'false).
 
     scramble_vertex_ids : bool
-    Flag controlling whether to scramble vertex ID bits (if set to `true`) or
-    not (if set to `false`); scrambling vertx ID bits breaks correlation
-    between vertex ID values and vertex degrees
+        Flag controlling whether to scramble vertex ID bits (if set to `true`)
+        or not (if set to `false`); scrambling vertx ID bits breaks correlation
```
Comment (rlratzel) [inline python/cugraph/generators/rmat.py:271]:
```suggestion
        or not (if set to `false`); scrambling vertex ID bits breaks correlation
```

File: python/cugraph/generators/rmat.py [hunk 4]
```diff
@@ -243,49 +243,68 @@ def rmat(
...
+        between vertex ID values and vertex degrees
```
Comment (rlratzel) [inline python/cugraph/generators/rmat.py:272]:
```suggestion
        between vertex ID values and vertex degrees.
```

**Review by rlratzel — CHANGES_REQUESTED** [review]
Just a few typos.

**Review by jnke2016 — COMMENTED** [review]
Nothing to add other than the few typo mentioned by Rick

**Review by rlratzel — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**